appmap/__init__.py

```diff
@@ -9,40 +22,42 @@
 if _enabled is None or _enabled.upper() == "TRUE":
     if _enabled is not None:
         # Use setdefault so tests can manage _APPMAP as necessary
-        os.environ["_APPMAP"] = _enabled
-    from _appmap import generation  # noqa: F401
-    from _appmap.env import Env  # noqa: F401
-    from _appmap.importer import instrument_module  # noqa: F401
-    from _appmap.labels import labels  # noqa: F401
-    from _appmap.noappmap import decorator as noappmap  # noqa: F401
-    from _appmap.recording import Recording  # noqa: F401
-
-    try:
-        from . import django  # noqa: F401
-    except ImportError:
-        pass
-
-    try:
-        from . import flask  # noqa: F401
-    except ImportError:
-        pass
-
-    try:
-        from . import fastapi  # noqa: F401
-    except ImportError:
-        pass
-
-    try:
-        from . import uvicorn  # noqa: F401
-    except ImportError:
-        pass
-
-    # Note: pytest integration is configured as a pytest plugin, so it doesn't
-    # need to be imported here
-
-    # unittest is part of the standard library, so it should always be
-    # importable (and therefore doesn't need to be in a try .. except block)
-    from . import unittest  # noqa: F401
-
-    def enabled():
-        return Env.current.enabled
+        os.environ.setdefault("_APPMAP", _enabled)
+        from _appmap import generation  # noqa: F401
+        from _appmap.env import Env  # noqa: F401
+        from _appmap.importer import instrument_module  # noqa: F401
+        from _appmap.labels import labels  # noqa: F401
+        from _appmap.noappmap import decorator as noappmap  # noqa: F401
+        from _appmap.recording import Recording  # noqa: F401
+
+        try:
+            from . import django  # noqa: F401
+        except ImportError:
+            pass
+
+        try:
+            from . import flask  # noqa: F401
+        except ImportError:
+            pass
+
+        try:
+            from . import fastapi  # noqa: F401
+        except ImportError:
+            pass
+
+        try:
+            from . import uvicorn  # noqa: F401
+        except ImportError:
+            pass
+
+        # Note: pytest integration is configured as a pytest plugin, so it doesn't
+        # need to be imported here
+
+        # unittest is part of the standard library, so it should always be
+        # importable (and therefore doesn't need to be in a try .. except block)
+        from . import unittest  # noqa: F401
+
+        def enabled():
+            return Env.current.enabled
+else:
+    print(_DISABLED_BY_DEFAULT_MSG, file=sys.stderr)
```

Comment:
Won't this cause the warning to be displayed even if the user explicitly sets `APPMAP=false`? 
This seems very annoying.

Reply:
Yeah, I noticed this was wrong right before I read your comment. I adjusted the indentation of the `else`, should work properly now.